File: April-Challenges-2021/Day-29/First-and-Last-Position-of-Elements-in-Sorted-Array.java
class Solution {
    public int[] searchRange(int[] nums, int target) {
        int[] ind = new int[2];
        ind[0] = -1;
        ind[1] = -1;
        if(nums.length==0){
            return ind;
        }
        for(int i = 0; i<nums.length; i++){
            if(nums[i]==target){
                ind[0] = i;
                break;
            }
        }
        if(ind[0]==-1){
            return ind;
        }
        for(int i = ind[0]; i<nums.length; i++){
            if(nums[i]==target){
                ind[1] = i;
            }
        }
        return ind;
    }
}

File: April-Challenges-2021/Day-29/First-and-Last-Position-of-Elements-in-Sorted-Array.kt
class Solution {
    fun searchRange(nums: IntArray, target: Int): IntArray {
        var ind = IntArray(2){-1;-1}
        for(i in nums.indices){
            if(nums[i]==target){
                ind[0] = i
                break
            }
        }
        if(ind[0]==-1){
            return ind
        }
        for(i in nums.indices){
            if(nums[i]==target){
                ind[1] = i
            }
        }
        return ind
    }
}
